### PR TITLE
pg gems and python modules need postgres libs to build

### PR DESCRIPTION
**Pull request by Kennric**
Adding postgresql dev libs for building pg gems and python modules. Kitchen converge errors on this repo, so not tested.

### CODE REVIEW

**Comment by ramereth** [comment]
@Kennric can you please add the pg-dev package in the serverspec tests too to ensure it's installed?

**Comment by Kennric** [comment]
@ramereth I think I did that correctly, but I am not getting kitchen to run: 
```
 » kitchen converge all 
>>>>>> 
No instances defined
```

File: spec/app1_spec.rb [hunk 1]
```diff
@@ -61,5 +61,9 @@
         expect(chef_run).to create_systemd_service(s)
       end
     end
+
+    describe package('postgresql-devel') do
+      it { should be_installed }
+    end
```
Comment (armiller) [inline spec/app1_spec.rb]: This is a serverspec test and needs to go within its own file in `test/integration`

File: spec/app1_spec.rb [hunk 2]
```diff
@@ -61,5 +61,4 @@
         expect(chef_run).to create_systemd_service(s)
       end
     end
-  end
```
Comment (armiller) [inline spec/app1_spec.rb]: This breaks the spec file

File: spec/app1_spec.rb (hunk 1 above)
Comment (Kennric) [inline spec/app1_spec.rb]: @armiller moved

**Comment by Kennric** [comment]
@armiller  ping

**Comment by armiller** [comment]
👍

**Comment by Kennric** [comment]
@ramereth one last look before I merge?

File: recipes/default.rb [hunk 3]
```diff
@@ -26,7 +26,8 @@
 
 # rvm package depends
 %w(sqlite-devel libyaml-devel readline-devel zlib-devel libffi-devel
-   openssl-devel automake libtool mariadb-devel ImageMagick-devel).each do |p|
+   openssl-devel automake libtool mariadb-devel ImageMagick-devel
```
Comment (ramereth) [inline recipes/default.rb]: Trailing space on the end

**Comment by Kennric** [comment]
@ramereth trailing space removed, not sure why my editor didn't strip that out.

**Comment by ramereth** [comment]
:+1:

**Comment by armiller** [comment]
@Kennric next time please use the standard `!bump patch` method described here https://docs.osuosl.org/config-management/chef/cookbooks/updating-organization-cookbooks.html

**Pull request merged**